Use universal-analytics to collect usage stats.

nya3jp opened the pull request.

hidehikoabe commented on src/index.js:
```diff
@@ -46,6 +48,10 @@ const App = () => (
 ReactDOM.render(<App />, document.getElementById('root'));
 
+const visitor =
+      UniversalAnalytics(constants.GOOGLE_ANALYTICS_TRACKING_ID, {https: true});
```
style: 4 indent.

hidehikoabe commented on src/constants.js:
```diff
@@ -26,3 +27,6 @@ export const PRESETS = [
   },
 ];
+
+// Google Analytics Tracking ID.
```
Optional: How about making this to the Jsdoc style? 

/*\* Google analytics Tracking ID */

hidehikoabe commented:
The change LGTM with minor comments.

BTW, what's the motivation to switch it from Google Analytics dedicated script to the third party one?
Could you clarify in the commit log?

nya3jp committed:
Use universal-analytics to collect usage stats.

After React adoption, Google Analytics script gives an error
on clicking the config button. I've not dug into details yet,
but it is likely caused by its page navigation hooks.

By migrating to universal-analytics module, we can control
when to submit UA events. Also we no longer need to specify
content security policies.

nya3jp committed:
Address comments.

nya3jp commented:
Addressed comments and added description. Thanks!
